homeassistant/components/metoffice/strings.json

```diff
@@ -7,13 +7,21 @@
           "api_key": "[%key:common::config_flow::data::api_key%]",
           "latitude": "[%key:common::config_flow::data::latitude%]",
           "longitude": "[%key:common::config_flow::data::longitude%]"
+        },
+        "data_description": {
+          "api_key": "Your DataHub API key",
+          "latitude": "Latitude of the weather site",
+          "longitude": "Longitude of the weather site"
```

Comment:
I think these are redundant and config flow looks cluttered with these but apparently they are required for `config_flow: done`...